src/app/modules/ix-tables/services/empty.service.ts

```diff
@@ -31,9 +39,16 @@ export class EmptyService {
         return this.loadingConfig;
       case EmptyType.Errors:
         return this.errorConfig;
+      case EmptyType.NoSearchResults:
+        return {
+          ...this.emptySearchResultsConfig,
+          title: this.translate.instant('No matching {title} found', { title }),
```

Comment:
Here and below we should just stick with generic message, similar to `No matching records found`.
This would work for English, but may not work in other languages, because word in `{title}` may need to be of different form when inside of `No matching {title} found` sentence.